packages/react-sdk/src/components/component-meta.ts

```diff
@@ -21,6 +21,7 @@ export const componentCategories = [
   "text",
   "media",
   "forms",
+  "hidden",
```

Comment:
Do we need it as special category? Omitted type worked well for us. All rich text child are hidden this way.

Reply:
thought we could improve ergonomics: 

`const { category, ...linkMetaRest } = linkMeta;` this is creating a local category variable we don't actually use, it works but seems more like a workaround